app/Http/Controllers/ProductController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Product;
+use App\Traits\Common;
+
+class ProductController extends Controller
+{
+    use Common;
+    /**
+     * Display a listing of the resource.
+     */
+    public function indexR()
+    {
+
+
+        $products = Product::orderBy('created_at', 'desc')->take(3)->get();
```

Comment:
good job ya Rasha,,

however,, there is a simpler function called `latest()` do the same as `orderBy('created_at', 'desc')` but your code still okay